xml/admin_monitoring_alerting.xml

```diff
@@ -1,1058 +1,374 @@
-       restart &prometheus;.
+        To tell the module to respond with “service unavailable”, set it to
+        <literal>fail</literal>:
       </para>
-     </example>
-    </listitem>
-   </itemizedlist>
-   <sect3 xml:id="alertmanager-templates">
-    <title>Templates</title>
-    <para>
-     You can use templates for label and annotation values. The
-     <varname>$labels</varname> variable includes the label key/value pairs of
-     an alert instance, while <varname>$value</varname> holds the evaluated
-     value of an alert instance.
-    </para>
-    <para>
-     The following example inserts a firing element label and value:
-    </para>
-<screen>
-{{ $labels.<replaceable>LABELNAME</replaceable> }}
-{{ $value }}
-</screen>
-   </sect3>
-   <sect3>
-    <title>Inspecting Alerts at Runtime</title>
-    <para>
-     If you need to verify which alerts are active, you have several options:
-    </para>
-    <itemizedlist>
-     <listitem>
+      <screen>&prompt.cephuser;ceph config set mgr mgr/prometheus/stale_cache_strategy fail</screen>
+    </sect2>
+    <sect2 xml:id="monitoring-rbd-image">
+      <title>Enabling RBD-image Monitoring</title>
       <para>
-       Navigate to the <guimenu>Alerts</guimenu> tab of &prometheus;. It will
-       show you the exact label sets for which defined alerts are active.
-       &prometheus; also stores synthetic time series for pending and firing
-       alerts. They have the following form:
+        The &prometheusmgr; can optionally collect RBD per-image IO statistics
+        by enabling dynamic OSD performance counters. The statistics are
+        gathered for all images in the pools that are specified in the
+        <literal>mgr/prometheus/rbd_stats_pools</literal> configuration
+        parameter.
+      </para>
+      <para>
+        The parameter is a comma or space separated list of
+        <literal>pool[/namespace]</literal> entries. If the namespace is not
+        specified the statistics are collected for all namespaces in the pool.
       </para>
-<screen>
-ALERTS{alertname="<replaceable>ALERT_NAME</replaceable>", alertstate="pending|firing", <replaceable>ADDITIONAL_ALERT_LABELS</replaceable>}
-</screen>
       <para>
-       The sample value is 1 if the alert is active (pending or firing). The
-       series is marked 'stale' when the alert is inactive.
+        For example:
       </para>
-     </listitem>
-     <listitem>
+      <screen>
+&prompt.cephuser;ceph config set mgr mgr/prometheus/rbd_stats_pools "<replaceable>pool1,pool2,poolN</replaceable>"
+      </screen>
       <para>
-       In the &prometheus; Web interface at the URL address
-       http://<replaceable>PROMETHEUS_HOST_IP</replaceable>:9090/alerts,
-       inspect alerts and their state (INACTIVE, PENDING or FIRING).
+        The module makes the list of all available images scanning the specified
+        pools and namespaces and refreshes it periodically. The period is
```

Comment:
Question, not suggestion:

Would this not make more sense if it said:

"The module scans the specified pools and namespaces and makes a list of all available images,
 and refreshes it periodically."
...?

Reply:
(It seems to me that the original phrasing implies that the images do the scanning, which does not make sense to me -- but I could be wrong.)